doc/run.md

```diff
@@ -342,7 +342,7 @@ packet egresses from the sender.
 --multi_src_port                     : debug option, use multiple src port for st20 tx stream.
 --audio_fifo_size <count>            : debug option, the audio fifo size between packet builder and pacing.
 --dhcp                               : debug option, enable DHCP for all ports.
---virtio_user                        : debug option, enable virtio_user ports for control plane packets.
+--virtio_user                        : debug option, enable virtio_user ports for control plane packets. Linux only, need root user.
```

Comment:
Why it need root?

Reply:
root privilege is required for operating tap device.

```
tap_open(): TUNSETIFF failed: Operation not permitted
virtio_user_dev_setup(): (/dev/vhost-net) Failed to setup backend
virtio_user_dev_init(): (/dev/vhost-net) backend set up fails
virtio_user_pmd_probe(): virtio_user_dev_init fails
EAL: Driver cannot attach the device (virtio_user0)
EAL: Failed to attach device on primary process
MT: Error: dev_if_init_virtio_user(0), cannot create virtio port
```

Reply:
ok, it may need some permission related settings.

Reply:
Yes, I found this works:
```
sudo setcap 'cap_net_admin+ep' ./build/app/RxTxApp
```